.github/workflows/stdlib-workflow-trigger.yml

```diff
@@ -0,0 +1,22 @@
+name: Stdlib Dependency Update
+
+on:
+  push:
```

Comment:
Also constrict only to master branch change

Reply:
Added with https://github.com/ballerina-platform/module-ballerina-websocket/pull/1/commits/e45e22eebc07bd7ac4a20f053f7c88b79e0614b0